recipes-bsp/bolt/bolt/boltctl

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# The purpose of this wrapper is to set LANG to UTF-8 so that non ASCII
+# characters in the output of boltctl can be displayed correctly
+LANG=en_US.UTF-8 /usr/libexec/boltctl "$@"
```

Comment:
You don't want double quotes around the `$@` in this case; right? I think that would eval to 
```
LANG=en_US.UTF-8 /usr/libexec/boltctl "$0 $1 $2 ..."
```

Reply:
[`$@` has special quoting rules](https://www.gnu.org/software/bash/manual/html_node/Special-Parameters.html#index-_0040)

Reply:
Oh. Didn't know that. :thinking: